codes/models/alt_growth_rates/ITA_adm2.do

```diff
@@ -43,25 +56,24 @@ lab var l_cum_confirmed_cases "log(cum_confirmed_cases)"
 gen D_l_cum_confirmed_cases = D.l_cum_confirmed_cases 
 lab var D_l_cum_confirmed_cases "change in log(cum_confirmed_cases)"
 
-// quality control
-replace D_l_cum_confirmed_cases = . if D_l_cum_confirmed_cases < 0 // cannot have negative changes in cumulative values
+// quality control: cannot have negative changes in cumulative values
+replace D_l_cum_confirmed_cases = . if D_l_cum_confirmed_cases < 0 
 
 
 //--------------testing regime changes
 
-gen testing_regime_change_feb26 = (t==21971)
+gen testing_regime_change_feb26 = (t==mdy(2,26,2020))
```

Comment:
Is there a reason we're hard coding testing regime changes in the regressions? (this actually applies to FRA I think and maybe some other countries too). If there is, feel free to just reply and/or resolve this comment. But if not, we should use the `testing_regime` variable for these, no?

Reply:
All of them are hard coded... It's because they only want to flag and control for the first day of the testing change (which is what this dummy does) instead of the entire period (which is what `testing_regime` does). But I can change the code to use `testing_regime` to create the first day dummy, so it's all programmatic.

Reply:
that would be great! Also, many of them have more than one testing_regime change right?

Reply:
some do. will create a separate pull request for creating these testing regime change dummies